### PR TITLE
Add Github Actions

### LINKED ISSUE #3: Create Github Actions for automatic linting and unit testing
Run pylint, flake8, and unit tests on PR to main

### PR DESCRIPTION
**Pull request by danyoungday**
Fixed #3 
Added linting via. pylint and flake8 as well as automatic unit testing to workflow.

### CODE REVIEW

File: .github/workflow/sdk.yml
Comment (danyoungday) [inline .github/workflow/sdk.yml:1]: A simpler version as from the MVP but with flake8 added

File: .pylintrc
Comment (danyoungday) [inline .pylintrc:1]: From the MVP as well

File: tests/test_dummy.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import unittest
+
+from src.prsdk.dummy import compute_percent_change
+
+
+class TestDummy(unittest.TestCase):
+    """
+    Tests for the dummy module.
+    """
+    def test_pct_change(self):
```
Comment (danyoungday) [inline tests/test_dummy.py:13]: A simple unit test to test the github actions

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+keyring==25.2.1
+markdown-it-py==3.0.0
+mccabe==0.7.0
+mdurl==0.1.2
+more-itertools==10.3.0
+nh3==0.2.18
+packaging==24.1
+pkginfo==1.10.0
+platformdirs==4.2.2
+prsdk==0.0.1
```
Comment (ofrancon) [inline requirements.txt]: We don't need all these requirements. Surely not `prsdk==0.0.1`
I think we only need:
```
flake8==7.1.0
pylint==3.2.6
```

**Review by ofrancon — CHANGES_REQUESTED** [review]
lgtm, except for the requirements.txt that should be trimmed to the libraries we really require

File: requirements.txt (hunk 2 above)
Comment (danyoungday) [reply requirements.txt]: Trimmed requirements down to these 2

**Review by ofrancon — APPROVED** [review]
Thanks!

**Pull request merged**